testing/benchmark/main.tf

```
@@ -20,6 +20,13 @@ locals {
     build        = var.BUILD_ID
     created_date = var.CREATED_DATE
   }
+
+  mandatory_tags = {
+    division  = "engineering"
+    org       = "obs"
+    team      = "apm-server"
+    project   = "apm-server"
```

```suggestion
    project   = "benchmarks"
```

I suggest we change this value to `benchmarks` to more clearly indicate its use.